Remotion/Web/Development.WebTesting/WebDriver/Factories/Chrome/ChromeBrowserFactory.cs

```diff
@@ -45,14 +47,25 @@ public IBrowserSession CreateBrowser (ITestInfrastructureConfiguration testInfra
     {
       ArgumentUtility.CheckNotNull ("testInfrastructureConfiguration", testInfrastructureConfiguration);
 
+      var securityWarningsEnabledRegistryHelper = new ChromiumSecurityWarningsRegistryHelper (_chromeConfiguration.DisableSecurityWarningsBehavior);
+      securityWarningsEnabledRegistryHelper.SetUp();

```

Comment:
As I'm looking at this, I actually believe that the stuff in SetUp() should be in the ctor. A ctor is meant to run setup code, running a `SetUp()` method after calling the ctor seems a little wrong. Calling `GetCleanupStrategy` does, on the other hand, seem fine. We need to unify if we camelcase Cleanup or not. I suggest we check other re-motion usages for that.

Reply:
> I suggest we check other re-motion usages for that.
`CleanUp` is a clear winner

Reply:
> As I'm looking at this, I actually believe that the stuff in SetUp() should be in the ctor. A ctor is meant to run setup code, running a SetUp() method after calling the ctor seems a little wrong. 

While I agree that the naming (`SetUp()`) is not great, the logic happening in this method should stay there and should not run in the `ctor`. Also, the "SetUp" should happen before the driver is created - having a separate method expresses the intent better than just having a `ctor` call.

Reply:
Check with MK

Reply:
I strongly prefer to have next to no logic in ctor that is not related to checking arguments. Having expensive or dependent code in ctor makes unit tests hard to write. 

The 'setup' concept is another matter. Do we even need a sepaeate setup concept or can we move this to GetCleanUpStrategy. Should this then be CreateCLeanUpStrategy and the Helper a Factory?

Reply:
`SetUp()` needs to be called before `CreateChromeDriver(...)`, which produces the userDirectory that is used by `ChromiumUserDirectoryCleanUp` and it makes sense that `ChromiumUserDirectoryCleanUp` and `GetCleanUpStrategy` are close together since they're both CleanUpStrategies, meaning `SetUp` and  `GetCleanUpStrategy()` cannot be together, like Romeo and Juliet. This problem might go away though, if we simply rename `SetUp` to something more fitting, but it's the registry set up the `RegistryHelper` does, so the name does fit in a way 🤷‍♂

Reply:
I don't see a conflict. Maybe if Juliet decided to visit Romeo in his bedchamber instead of the other way 'round they would be able to live happily ever after? Provided she's not found out by her parents and sent to a convent.